available-connectors.json

```diff
@@ -191,6 +201,28 @@
     "supportedByStrongLoop": true
   },
   {
+    "name": "kv-redis",
+    "description": "Redis key-value connector",
+    "baseModel": "KeyValueModel",
+    "features": {
+      "discovery": false,
+      "migration": false
+    },
+    "settings": {
+      "host": {
+        "type": "string"
+      },
+      "port": {
+        "type": "number"
+      }
```

Comment:
@superkhau Could you please point me to a place where `user` and `pass` are documented? AFAICT from https://github.com/luin/ioredis/blob/master/API.md#new_Redis_new, the only authentication-related setting is `password`.